phpunit/class-wp-rest-post-format-search-handler-test.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * WP_REST_Post_Format_Search_Handler tests
+ *
+ * @package Gutenberg
+ */
+
+/**
+ * Tests for WP_REST_Post_Format_Search_Handler_Test.
+ *
+ * @group restapi
+ */
+class WP_REST_Post_Format_Search_Handler_Test extends WP_Test_REST_Controller_Testcase {
+
+	/**
+	 * Term associated with a post format.
+	 *
+	 * @var array
+	 */
+	private static $my_post_format_term = array();
+
+	/**
+	 * Create fake data before our tests run.
+	 *
+	 * @param WP_UnitTest_Factory $factory Helper that lets us create fake data.
+	 */
+	public static function wpSetUpBeforeClass( $factory ) {
+		self::$my_post_format_term = $factory->term->create(
```

Comment:
Can we use `set_post_format` on a created post instead? That feels more durable.

Reply:
Done in 26e1cb0173.